.github/action/create-env.py

```diff
@@ -4,12 +4,19 @@
 
 out_file = ""
 
+filepath = os.path.join(
+    str(os.environ.get("GITHUB_WORKSPACE")), str(os.environ.get("FILE_NAME"))
+)
+
 for key in env_keys:
     if key.startswith("ENVKEY_"):
         out_file += key.split("ENVKEY_")[1] + "=" + os.environ.get(key) + "\n"
 
 with open(str(os.environ.get("GITHUB_WORKSPACE")) + "/" + str(os.environ.get("FILE_NAME")), "w") as text_file:
     text_file.write(out_file)
+filepath
```

Comment:
```suggestion
filepath
```
We need to remove this from vagrant-hashistack code as well.